pandas/io/excel/_base.py

```diff
@@ -700,7 +700,14 @@ class ExcelWriter(metaclass=abc.ABCMeta):
 
         .. versionadded:: 1.3.0
     engine_kwargs : dict, optional
-        Keyword arguments to be passed into the engine.
+        Keyword arguments to be passed into the engine. These will be passed to
+        the following functions of the respective engines:
+
+        * odswriter: ``odf.opendocument.OpenDocumentSpreadsheet(**engine_kwargs)``
```

Comment:
can you make these in order of importance (xlswwrite, openpyxl) and then odfwriter, or alphabetic.

remove xlwt